luxtronik/datatypes.py

```diff
@@ -472,6 +472,14 @@ class HeatpumpCode(SelectionBase):
         29: "L2H",
         30: "WZWD",
         31: "ERC",
+        32: "ERC",
+        33: "ERC",
+        34: "ERC",
+        35: "ERC",
+        36: "ERC",
+        37: "ERC",
+        38: "ERC",
+        39: "ERC",
```

Comment:
Is this really 8 time the same code?

Reply:
Yes, this is what you would see in the luxtronik controller for a heat pump of type 31-39. I am not sure, if one can find any of these in the wild...

Reply:
Does not sound unfamiliar for AIT to do such a mess 😅 I was just curious if it was kind of a typo/copy and paste error

Reply:
What is `ERC`? Does anyone know?

Reply:
No idea. Maybe this is just some leftover...

Reply:
I've looked around and every source I found says something like ERC = unknown. I would merge this and if some day in the future the real meaning comes to light we can fix that.